packages/effect/src/Metric.ts

```diff
@@ -45,7 +45,7 @@ export type MetricTypeId = typeof MetricTypeId
  *   - Histograms
  *   - Summaries
  *
- * @since 2.0.0
+ * @since 3.6.5
```

Comment:
The since tags would stay unchanged, as they reflect when something was added rather than modified.

Reply:
@tim-smart - reverted the since tags that did not reflect additions.